activerecord/test/cases/base_test.rb

```diff
@@ -1495,4 +1495,12 @@ def test_default_values_are_deeply_dupped
     # regular column
     assert query.include?("name")
   end
+
+  test "column names are quoted when using #from clause and model has ignored columns" do
+    refute_empty Developer.ignored_columns
+    query = Developer.from("`developers`").to_sql
+    quoted_id = Developer.connection.quote_table_name("id")
+
+    assert_match /SELECT #{quoted_id}.* FROM `developers`/, query
```

Comment:
This generates a ruby warning. We need a `()` around the arguments. Also can we hardcode `quoted_id` instead of using a variable? It will make the test easier to understand.

Reply:
Sounds good for the `()` 

I added the dynamic `quoted_id` because based on the adapter the quoting is different. MySQL uses backticks while other uses simple `"`. I can make a conditional if you prefer

Reply:
Oh. That is true. Let's only add the `()`